Testing/SystemTests/tests/framework/PowderReduceP2DTest.py

```diff
@@ -67,11 +65,22 @@ def runTest(self):
             SystemTest=True,
         )
 
-    def validateMethod(self):
-        return "ValidateAscii"
+    def doValidation(self):
```

Comment:
Please add a docstring/comment on this function that highlights it is a custom validation and the `validate()` method is being overridden to reflect that